prow/external-plugins/cherrypicker/server.go

```diff
@@ -390,8 +391,26 @@ func (s *Server) handle(l *logrus.Entry, requestor string, comment *github.Issue
 		return err
 	}
 
-	// Checkout a new branch for the cherry-pick.
+	// New branch for the cherry-pick.
 	newBranch := fmt.Sprintf(cherryPickBranchFmt, num, targetBranch)
+
+	// Check if that branch already exists, which means there is already a PR for that cherry-pick.
```

Comment:
would it make sense instead to check for a commit than the branch name?

Reply:
I chose the branch, because that's exactly what triggers the error message if we try to make a PR for the cherrypick.